Dockerfile

```diff
@@ -16,32 +18,21 @@ RUN apt-get update && \
                        python-numpy \
                        python-toml \
                        python-yaml \
-                       unzip
-
-RUN curl -ssL http://get.gazebosim.org > /tmp/install_gazebosim.sh
-RUN sed -i "s/GZ_VER=9/GZ_VER=8/" /tmp/install_gazebosim.sh
-RUN chmod +x /tmp/install_gazebosim.sh
-RUN /tmp/install_gazebosim.sh
+                       unzip \
+                       gazebo9 \
+                       libgazebo9-dev
 
 RUN git clone --depth 1 https://github.com/PX4/Firmware.git ${FIRMWARE_DIR}
 RUN git -C ${FIRMWARE_DIR} submodule update --init --recursive
 
-# Initial location of the drone
```

Comment:
Why removing those?

Reply:
See commits messages.

Reply:
Argh, I always do that and rely on the PR description :sweat_smile: